com.unity.shadergraph/Editor/Drawing/Controllers/BlackboardController.cs

```diff
@@ -175,6 +175,7 @@ void CopyShaderInput(GraphData graphData)
             {
                 case AbstractShaderProperty property:
 
+					insertIndex = Mathf.Clamp(insertIndex, -1, graphData.properties.Count() - 1);
```

Comment:
noticed that this could in some cases overflow past the size of the property/keyword/dropdown arrays, and `AddCopyOfShaderInput` does not account for that so clamping here